tests/test_api_push_validate.py

```diff
@@ -396,17 +396,6 @@ def test_10_enroll_push(self):
         # Set Policy scope:auth, action:enroll_via_multichallenge=push
         set_policy("pol_multienroll", scope=SCOPE.AUTH,
                    action="{0!s}=push".format(ACTION.ENROLL_VIA_MULTICHALLENGE))
-        # We still do a passthru, since we do not have a firebase config, yet.
-        with self.app.test_request_context('/validate/check',
```

Comment:
I remove this test, since it only occurred in an not complete configuration.
We now also raise an error, if the firebase config is missing, which also makes more sense.